lib/freight/conf.sh

```diff
@@ -6,18 +8,23 @@ VARLIB="/var/lib/freight"
 VARCACHE="/var/cache/freight"
 
 # Default architectures.
+# shellcheck disable=SC2034
```

Comment:
that doesn't work for me, I think koalaman/shellcheck#495 is the RFE for similar functionality